src/ui/components/achievements/components/GenericAchievementCard.tsx

```diff
@@ -116,14 +116,16 @@ Those props are provided by the GenericAchievementCard and must be passed down t
           </FlexContainer>
         )}
       </BottomButtonsContainer>
-      <Spacer.Flex flex={2} />
+      <Spacer.Flex
+        flex={props.lastIndex !== 0 ? 2 : getGrid({ default: 0, sm: 0.25 }, Axis.HEIGHT)}
```

Comment:
C'est possible de faire comme cela ? Car tu as utilisé cette condition pour le FlexContainer au-dessus
```suggestion
        flex={!props.lastIndex ? 2 : getGrid({ default: 0, sm: 0.25 }, Axis.HEIGHT)}
```